proto/buf/alpha/lint/v1/config.proto

```diff
@@ -0,0 +1,58 @@
+// Copyright 2020-2021 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package buf.alpha.lint.v1;
+
+// Config represents the lint configuration for a module. The rule and category IDs are defined
+// by the version and apply across the config. The version is independent of the version of
+// the package. The package version refers to the config shape, the version encoded in the Config message
+// indicates which rule and category IDs should be used.
+//
+// The rule and category IDs are not encoded as enums in this package because we may want to support custom rule
+// and category IDs in the future. Callers will need to resolve the rule and category ID strings.
+message Config {
+  // version represents the version of the lint rule and category IDs that should be used with this config.
+  string version = 1;
+  // use_rule_ids lists the rule and/or category IDs that are included in the lint check.
+  repeated string use_rule_ids = 2;
+  // except_rule_ids lists the rule and/or category IDs that are excluded from the lint check.
+  repeated string except_rule_ids = 3;
```

Comment:
According to the edits above, don't forget these.

```suggestion
  // use_ids lists the rule and/or category IDs that are included in the lint check.
  repeated string use_ids = 2;
  // except_ids lists the rule and/or category IDs that are excluded from the lint check.
  repeated string except_ids = 3;
```

Reply:
😳